civis/parallel.py

```diff
@@ -710,7 +710,7 @@ def _fetch_result(fut):
 
             try:
                 # Find the output file ID from the run outputs.
-                run_outputs = client.scripts.list_containers_runs_outputs(
+                run_outputs = client.jobs.list_runs_outputs(
```

Comment:
This `_CivisBackendResult` class is designed to work with only _container_ scripts, so I don't think we should modify `_CivisBackendResult`'s implementation.

If you take back this change and have both `scripts.list_containers_runs_outputs` and `jobs.list_runs_outputs` mocked out at the `create_client_mock_for_container_tests` function below, then the test suite will pass.